sensor/models.py

```diff
@@ -209,18 +209,22 @@ def save(self, *args, **kwargs):
     def error(cls, data):
         if data is None:
             return "Error: empty payload"
-
+    
```

Comment:
Please no trailing whitespace, so please run `M-x whitespace-cleanup` on the files, or `git rebase --whitespace=fix HEAD~1` or something on these files.